bridge/deploy/21_deploy_allowTokens.js

```diff
@@ -172,11 +208,11 @@ async function setTokensRskTestnet(allowTokens) {
         { token: '0x4cfE225cE54c6609a525768b13F7d87432358C57', typeId: '4' }, //rKovUSDT
         { token: '0x8bbbd80981fe76d44854d8df305e8985c19f0e78', typeId: '3' }, //rKovLINK
         { token: '0xe95afdfec031f7b9cd942eb7e60f053fb605dfcd', typeId: '3' }, //rKovsBUND
-    ]);
+    ]).send({ from: deployer });
 }
 
-async function setTokensKovan(allowTokens) {
-    await allowTokens.setMultipleTokens([
+async function setTokensKovan(allowTokens, deployer) {
+    await allowTokens.methods.setMultipleTokens([
```

Comment:
why is it now necessary to use the methods member? is this because hardhat doesn't add the ABI to the contract object or something? would this affect our tests?

Reply:
web3 uses the methods member, truffle hides it but it's just truffle, as we are using hardhat we don't have that any more at deploy